hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/volume/TestOMVolumeResponse.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.hadoop.ozone.om.response.volume;
```

Comment:
Please add the license header.

Reply:
Thank you for your review. I have added the license and resolved the checkstyle issues.